resources/definitions/mixware_wand.def.json

```diff
@@ -0,0 +1,194 @@
+{
+  "name": "Wand",
+  "version": 2,
+  "inherits": "fdmprinter",
+  "metadata": {
+    "visible": true,
+    "author": "Mixware",
+    "manufacturer": "Mixware",
+    "file_formats": "text/x-gcode",
+    "has_materials": true,
+    "has_machine_quality": true,
+    "preferred_material": "mixware_pla",
+    "preferred_quality_type": "coarse",
+    "has_variants": false,
+    "platform": "mixware_wand_platform.STL",
+    "machine_extruder_trains": {
+      "0": "mixware_wand_extruder_0"
+    }
+  },
+  "overrides": {
+    "machine_name": {
+      "default_value": "Wand"
+    },
+    "machine_start_gcode": {
+      "default_value": "G28 ;Home\n;Prime the extruder\nM117 Print\nG92 E0\nG1 F1500 X-2 Y0 Z0.28\nG1 F2400 X-2 Y70 Z0.28 E8\nG1 F1500 X-1.7 Y70 Z0.28\nG1 F2400 X-1.7 Y0 Z0.28 E16\nG92 E0\nG92 E0\nG1 Z0.18 F2400 ;Move Z Axis up"
+    },
+    "machine_end_gcode": {
+      "default_value": "M104 S0\nM140 S0\n;Retract the filament\nG92 E1\nG1 E3 F300\nG1 F1500 Z100\nG28 X0 Y0\nM25\nM84"
+    },
+    "material_diameter": {
+      "default_value": 1.75
+    },
+    "machine_width": {
+      "default_value": 102
+    },
+    "machine_depth": {
+      "default_value": 102
+    },
+    "machine_height": {
+      "default_value": 102
+    },
+    "machine_nozzle_size": {
+      "default_value": 0.4
+    },
+    "layer_height": {
+      "minimum_value": "0.05",
+      "maximum_value": "0.2"
+    },
+    "wall_thickness": {
+      "default_value": 0.8,
+      "minimum_value": "0.4",
+      "maximum_value": "1.6"
```

Comment:
Aare you sure about that? Maximum_value means that it will prevent you from slicing with it. I assume that you intended to set a warning?